### ISSUE TITLE
DELETE method lost parameters

### ISSUE THREAD

**Issue by weicong** [kubb-project/kubb#186]
https://demo.aspnetzero.com/swagger/v1/swagger.json

```json
    "/api/services/app/User/DeleteUser": {
      "delete": {
        "tags": [
          "User"
        ],
        "operationId": "DeleteUser",
        "parameters": [
          {
            "name": "id",
            "in": "query",
            "schema": {
              "type": "integer",
              "format": "int64"
            }
          }
        ],
        "responses": {
          "200": {
            "description": "Success"
          }
        }
      }
    },
```

```ts
import client from "../../../utils/axios-client";
import type { DeleteUserRequest, DeleteUserResponse } from "../../models/User/DeleteUser";

        /**
* @link /api/services/app/User/DeleteUser
*/
        export function deleteUser <TData = DeleteUserResponse, TVariables = DeleteUserRequest>() {
          return client<TData, TVariables>({
            method: "delete",
            url: `/api/services/app/User/DeleteUser`
            params, // missing !!!
          });
        };
```

DeleteUser method lost 
```json
        "parameters": [
          {
            "name": "id",
            "in": "query",
            "schema": {
              "type": "integer",
              "format": "int64"
            }
          }
        ],
```

**Comment by weicong** [comment]
The DELETE method should be similar to the GET method

```ts
import type { DeleteUserResponse, DeleteUserQueryParams  } from "../../models/User/DeleteUser";

export type DeleteUserQueryParams = {
  /**
  * @type integer | undefined int64
  */
  id?: number | undefined;
};

        /**
* @link /api/services/app/User/DeleteUser
*/
        export function deleteUser <TData = DeleteUserResponse, TParams = DeleteUserQueryParams>( params?: TParams) {
          return client<TData>({
            method: "delete",
            url: `/api/services/app/User/DeleteUser`,
            params,
          });
        };
```

GET:
```ts
import type { GetUserForEditResponse, GetUserForEditQueryParams } from "../../models/User/GetUserForEdit";

      /**
* @link /api/services/app/User/GetUserForEdit
*/
      export function getUserForEdit <TData = GetUserForEditResponse, TParams = GetUserForEditQueryParams>( params?: TParams) {
        return client<TData>({
          method: "get",
          url: `/api/services/app/User/GetUserForEdit`,
          params,
        });
      };
```

**Comment by stijnvanhulle** [comment]
Will be included in `@kubb/core` v 1.0.0-beta.11
